sql-statements/sql-statement-select.md

```diff
@@ -144,6 +148,7 @@ mysql> SELECT * FROM t1;
 - The syntax `SELECT ... INTO @variable` is not supported.
 - The syntax `SELECT ... GROUP BY ... WITH ROLLUP` is not supported.
 - The syntax `SELECT .. GROUP BY expr` does not imply `GROUP BY expr ORDER BY expr` as it does in MySQL 5.7. TiDB instead matches the behavior of MySQL 8.0 and does not imply a default order.
+- The syntax `SELECT ... TABLESAMPLE ...` is a TiDB extension and not supported by MySQL.
```

Comment:
Since this syntax is TiDB only. Do you think providing an example in `## Examples` will be useful for users?

Reply:
Looks good to me. Would you please add this example to the doc? Thanks.